cvprac/cvp_api.py

```diff
@@ -246,20 +246,19 @@ def get_inventory(self, start=0, end=0):
                              (start, end), timeout=self.request_timeout)
         return data['netElementList']
 
-    def get_device_container_map(self, start=0, end=0):
-        ''' Returns the a dict of the device to parent container mapping.
+    def get_devices_in_container(self, name):
+        ''' Returns a dict of the devices under the named container.
 
             Args:
-                start (int): The first inventory entry to return.  Default is 0
-                end (int): The last inventory entry to return.  Default is 0
-                    which means to return all inventory entries.  Can be a
-                    large number to indicate the last inventory entry.
+                name (str): The name of the container to get devices from
         '''
-        self.log.debug('get_device_container_map: called')
+        self.log.debug('get_devices_in_container: called')
         data = self.clnt.get('/inventory/getInventory.do?'
-                             'queryparam=&startIndex=%d&endIndex=%d' %
-                             (start, end), timeout=self.request_timeout)
-        return data['containerList']
+                             'queryparam=%s&startIndex=0&endIndex=0' % name,
+                             timeout=self.request_timeout)
+        if data['total'] > 0:
+            return data['containerList']
```

Comment:
Per our discussion I think it would be more helpful to return the netElementList here.